snapshot/bundlestore/README.md

```diff
@@ -10,13 +10,33 @@ There are a few different store types that handle bundles differently. They can
 in memory, bundles on disk, and bundles located behind an http server. Further, we can
 wrap these stores in a parent store to add additional business logic to our handling.
 
+## Bazel Content Addressable Store
+Bundlestore servers now also support the CAS API over gRPC, backed by the same underlying Store as the HTTP API.
+
 ## Bundle name conventions
-For now names look like 'bs-<sha>.bundle'.
+* HTTP Bundlestore server - For now names look like 'bs-<sha>.bundle'
+* GRPC Bazel CAS server - based on Digest - see Bazel remote execution/CAS documentation
+Note that the above means that although the Store backing a server can support HTTP and GRPC APIs,
+the HTTP API naming restriction prevents accesing GRPC artifacts.
```

Comment:
sp: accessing. I'm not sure what the implications of this are / why we can't check for 40 or 64 digits and act accordingly.

Reply:
this was mainly for by-hand testing convenience, e.g. write with grpc then read with http. the APIs are not really cross-compatible for other reasons, though, so I left out naming derestriction for http until there is a real reason to have it